TodoReview.py

```diff
@@ -150,6 +150,8 @@ def run(self, edit, **args):
 		window = self.view.window()
 		paths = args.get('paths', None)
 		settings = Settings(self.view, args.get('settings', False))
+		if not paths and not settings.get('current_file', False):
```

Comment:
I think this needs to be `args.get` rather than `settings.get` just based on line 151 and 155. The settings object will grab from the JSON config while the args object gets specific run-time parameters, like from the keyboard config you referenced.